src/GuzzleClientFactory.php

```diff
@@ -68,7 +68,7 @@ public function getClient(string $baseUrl, array $options = []): GuzzleClient
             $messages = [];
             /** @var ConstraintViolationInterface $error */
             foreach ($errors as $error) {
-                $messages[] = sprintf('Value "%s" is invalid: %s', $error->getInvalidValue(), (string) $error->getMessage());
+                $messages[] = sprintf('Options error: %s', $error->getMessage());
```

Comment:
cim je to motivovany? nemuze se stat, ze to nekde checkujeme v catchi? Zevrubne jsem se koukal a nic jsemn nenasel, ale jen se ptam, na duvod.

Reply:
Invalid value je teď mixed takže nejde udělat cast to string jednoduše. Musel bych to hákovat a to je zbytečné, protože je to libka pro connection co umře a jestli vrací validace trochu jinů hlášku je fuk.